ocs_ci/ocs/hsbench.py

```diff
@@ -134,35 +152,202 @@ def run_benchmark(
+                        )
+                else:
+                    log.warning(f"No objects data to validate in bucket {bucket_name}")
+
+    def validate_hsbench_put_get_list_objects(
+        self, result=None, num_objs=None, put=None, get=None, list_obj=None
+    ):
+        """
+        Validate PUT, GET, LIST objects from previous hsbench operation
+
+        Agr:
+            result (str): Result file name
+            num_objs (str): Number of objects to validate
+            put (Boolean): Validate PUT operation
+            get (Boolean): Validate GET operation
+            list_obj (Boolean): Validate LIST operation
+
+        """
+        eval_data = {}
+        with open(f"{self.warp_dir}/{result}", "r") as file, open(
+            f"{self.warp_dir}/summary.csv", "w"
+        ) as out:
+            writer = csv.writer(out)
+            for row in csv.reader(file):
+                if row[1] == "TOTAL":
+                    writer.writerow(row)
+
+        with open(f"{self.warp_dir}/summary.csv", "r") as read_obj:
+            reader = csv.reader(read_obj)
+            for row in reader:
+                eval_data[row[3]] = row[4]
+
+        if put is True:
+            put_val = eval_data["PUT"]
+            log.info(f"Number of PUT objects is {put_val}")
+            if put_val != num_objs:
+                assert put_val, (
+                    "Number of PUT objects don't match with number objects."
+                    f"Expecting {num_objs} but getting {put_val}"
+                )
+        if get is True:
+            get_val = eval_data["GET"]
+            log.info(f"Number of GET objects is {get_val}")
+            if get_val != num_objs:
+                assert get_val, (
+                    "Number of GET objects don't match with number objects."
+                    f"Expecting {num_objs} but getting {get_val}"
+                )
+        if list_obj is True:
+            list_val = eval_data["LIST"]
+            log.info(f"Number of LIST objects is {list_val}")
+            if list_val != 0:
+                assert list_val, (
+                    "LIST objects doesn't show correctly."
+                    f"Expecting LIST objects is not zero but getting {list_val}"
+                )
+
+    def delete_objects_in_bucket(self, bucket_name=None):
+        """
+        Delete objects in a bucket
+
+        Agr:
+            bucket_name (str): Name of bucket
+
+        """
+        log.info("Deleting objects in bucket...")
+        self.toolbox.exec_sh_cmd_on_pod(
+            f"radosgw-admin object rm --object=000000000000 --bucket={bucket_name}"
+        )
+        num_object = self.toolbox.exec_sh_cmd_on_pod(
+            f"radosgw-admin bucket stats --bucket={bucket_name} | grep num_objects"
+        )
+        log.info(f"Number objects in {bucket_name} is {num_object}")
+        if num_object == (self.num_obj - 1):
```

Comment:
This function deletes all objects in the bucket? If so checking existing object after delete is not necessary.

Reply:
This function is only deleting 1 object "--object=000000000000" in bucket.  The rest of the objects are still usable.
https://github.com/red-hat-storage/ocs-ci/blob/1332ebb63dd97a2a51c7556fb8221f959740e2ee/ocs_ci/ocs/hsbench.py#L315